src/nodes/display/ToneMappingNode.js

```diff
@@ -11,30 +11,37 @@ import { NoToneMapping, LinearToneMapping, ReinhardToneMapping, CineonToneMappin
 // exposure only
 const LinearToneMappingNode = Fn( ( { color, exposure } ) => {
 
-	return color.mul( exposure ).clamp();
+	const colortone = vec3( color ).mul( exposure );
+
+	colortone.assign( clamp( colortone, 0.0, 1.0 ) );
+
+	return vec4( colortone, color.a );
```

Comment:
@sunag I have started to rewrite these functions, and would appreciate some feedback. Is there a better way to write these?

I am trying to follow the same pattern for each function.